redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/DatabaseServerV4Endpoint.java

```diff
@@ -52,25 +52,32 @@ DatabaseServerV4Responses list(@NotNull @QueryParam("environmentId") String envi
     @Path("{nameOrCrn}")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = DatabaseServerOpDescription.GET_BY_NAME_OR_CRN, produces = MediaType.APPLICATION_JSON, notes = Notes.DATABASE_SERVER_NOTES,
-            nickname = "getDatabaseServer")
+            nickname = "getDatabaseServerByNameOrCrn")
     DatabaseServerV4Response getByNameOrCrn(@NotNull @QueryParam("environmentId") String environmentId, @PathParam("nameOrCrn") String nameOrCrn);
 
+    @GET
+    @Path("/crn/{crn}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = DatabaseServerOpDescription.GET_BY_CRN, produces = MediaType.APPLICATION_JSON, notes = Notes.DATABASE_SERVER_NOTES,
+            nickname = "getDatabaseServerByCrn")
+    DatabaseServerV4Response getByCrn(@PathParam("crn") String crn);
+
     @POST
-    @Path("external")
+    @Path("managed")
```

Comment:
I know I said I'm bad at naming things, but I'll still say that I don't like "managed" here, because it doesn't say _who_ is managing - is it redbeams or something else? At least "external" means "outside", which seems clear to me is not redbeams.

Reply:
I see. What about serviceManaged?

External to me is confusing, it could mean:
1) external in relation to redbeams, because the user manages the lifecycle of it, redbeams only uses it
2) external in relation to the user, redbeams manages it.

Reply:
I see you have the same confusion with "external" that I have with "managed". :)

To me, "serviceManaged" doesn't help - which service? Redbeams?